recipes/systemc/meta.yaml

```
@@ -0,0 +1,118 @@
+# systemc-static just the static library
+# systemc-doc  documentation including the installed examples
+
+outputs:                                                         # [not win]
+    - name: libsystemc                                           # [not win]
+      requirements:                                              # [not win]
+          build:                                                 # [not win]
+              - {{ compiler('cxx') }}                            # [not win]
+      files:                                                     # [not win]
+          - lib/libsystemc-*                                     # [not win]
+      test:                                                      # [not win]
+        commands:                                                # [not win]
+          - test -f "$PREFIX"/lib/libsystemc-{{ version }}"$SHLIB_EXT"  # [not win]
+    - name: libsystemc-static                                    # [not win]
+      files:                                                     # [not win]
+          - lib/libsystemc.a                                     # [not win]
+    - name: systemc-doc                                          # [not win]
+      files:                                                     # [not win]
+          - share/doc/systemc/                                   # [not win]
+    - name: systemc-dev                                          # [not win]
+      requirements:                                              # [not win]
+          build:                                                 # [not win]
+              - {{ compiler('cxx') }}                            # [not win]
+          run:                                                   # [not win]
+              - {{ pin_subpackage('libsystemc', exact=True) }}   # [not win]
+      files:                                                     # [not win]
+          - lib/libsystemc.so                                    # [not win]
+          - lib/pkgconfig/systemc.pc                             # [not win]
+          - lib/pkgconfig/tlm.pc                                 # [not win]
+          - include/sysc                                         # [not win]
+          - include/systemc*                                     # [not win]
+          - include/tlm*                                         # [not win]
+          - include/tlm_core                                     # [not win]
+          - include/tlm_utils                                    # [not win]
+          - etc/conda/*activate.d/*activate-systemc.sh           # [not win]
+      test:                                                      # [not win]
+        requires:                                                # [not win]
+          - {{ compiler('cxx') }}                                # [not win]
+          - {{ pin_subpackage('systemc-doc', exact=True) }}      # [not win]
+          - make                                                 # [not win]
```

```suggestion
          - make                                                 # [not win]
          - sed                                                    # [not win]
```
also install cf GNU sed because the sed on os-x is wonk